src/Microsoft.VisualStudio.Threading.Tests/TplExtensionsTests.cs

```diff
@@ -151,12 +151,47 @@ public void ApplyResultToPreCompletedNonGeneric()
         [Fact]
         public void WaitWithoutInlining()
```

Comment:
Do we need a test for WaitWithoutInlining on the tasks that:
1. Have other tasks attached to parent
2. Failed or cancelled